fileglancer/proxiedpath.py

```diff
@@ -0,0 +1,98 @@
+import requests
+import logging
+
+from typing import Optional
+from functools import cache
+
+from .uimodels import ProxiedPath, CachedEntry
+
+
+log = logging.getLogger(__name__)
+
+
+class ProxiedPathManager:
+    """Manage the list of user shared data paths from the central server."""
+
+    def __init__(self, central_url: str):
+        self.central_url = central_url
+        self._cached_proxied_paths = {}
+    
+    def get_proxied_paths(self, username: str, sharing_key: Optional[str] = None) -> list[ProxiedPath]:
+        """Lookup a user shared path by its sharing key."""
+        cached_user_proxied_paths = self._cached_proxied_paths.get(username)
```

Comment:
I'm not sure we need to cache these proxied paths at all. The reason we cache the file share paths is because there was no way to get just one (maybe that should be added). But for proxied paths, we have full CRUD. Is there any other reason to cache these here?

Reply:
No reason at all - it was for consistency with the paths manager but I can remove it